### ISSUE TITLE
Showing not safe when everything is safe

### ISSUE THREAD

**Issue by pchote** [warwick-one-metre/environmentd#10]
```
It is NOT SAFE to observe

Vaisala data from 19:17:24 — 19:37:19:
             Wind: 6.1 — 22.2 km/h
         Pressure: 779.9 — 780.0 hPa
    Outside Temp.: 11.3 — 11.6℃
     Outside Hum.: 21.6 — 23.6 %RH

RoomAlert data from 19:17:27 — 19:37:13:
   Internal Temp.: 10.3 — 11.5℃
    Internal Hum.: 18.04 — 21.24 %RH
      Truss Temp.: 12.12 — 14.69℃
     Server Temp.: 14.1 — 16.1℃
      Server Hum.: 11.17 — 14.09 %RH
       Side Hatch: CLOSED
        Trap Door: CLOSED

SuperWASP data from 19:17:28 — 19:37:13:
     Outside Hum.: 19.0 — 21.0 %RH
        Sky Temp.: -48.3 — -41.0℃
  Temp. > Dew Pt.: 21.32 — 22.67℃

Disk space at 19:18:49:
   Data Partition: 29 GiB (27%) available

saft@sqftnuc2:~> environment json
{"roomalert_truss_temp": [12.12, 14.69, true], "superwasp_ext_humidity": [19.0, 21.0, true], "superwasp_dew_point_delta": [21.32, 22.67, true], "roomalert_queue_start": "2016-02-13T19:13:41Z", "roomalert_measurement_end": "2016-02-13T19:37:13Z", "roomalert_trap_closed": [true], "vaisala_wind_speed": [6.1, 22.2, true], "vaisala_queue_len": 132, "vaisala_sufficient_data": true, "superwasp_queue_start": "2016-02-13T19:14:42Z", "vaisala_pressure": [779.9, 780.0, true], "diskspace_data_fs_percent_available": [27, true], "superwasp_queue_len": 132, "roomalert_measurement_start": "2016-02-13T19:17:27Z", "vaisala_queue_start": "2016-02-13T19:15:24Z", "diskspace_measurement_start": "2016-02-13T19:18:49Z", "vaisala_temperature": [11.3, 11.6, true], "roomalert_internal_humidity": [18.04, 21.24, true], "diskspace_queue_start": "2016-02-13T19:12:48Z", "roomalert_hatch_closed": [true], "roomalert_roomalert_humidity": [11.17, 14.09, true], "vaisala_measurement_start": "2016-02-13T19:17:34Z", "superwasp_measurement_end": "2016-02-13T19:37:23Z", "diskspace_data_fs_available_bytes": [31602171904, true], "roomalert_sufficient_data": true, "roomalert_roomalert_temp": [14.1, 16.1, true], "superwasp_sufficient_data": true, "vaisala_measurement_end": "2016-02-13T19:37:19Z", "diskspace_sufficient_data": false, "superwasp_sky_temp": [-48.3, -41.0, true], "superwasp_measurement_start": "2016-02-13T19:17:28Z", "can_observe": false, "vaisala_relative_humidity": [21.6, 23.6, true], "diskspace_measurement_end": "2016-02-13T19:18:49Z", "roomalert_queue_len": 132, "diskspace_queue_len": 4, "roomalert_internal_temp": [10.3, 11.5, true]}
```

![screen shot 2016-02-13 at 20 07 09](https://cloud.githubusercontent.com/assets/167819/13029773/68a19758-d28d-11e5-9879-ddb9310bb3d3.png)

**Comment by pchote** [comment]
`"diskspace_sufficient_data": false,`

**Comment by pchote** [comment]
Fixed by cbc04640672872e5a69e7186e80fbff4a5def61e, 89a78d980233cd49fee5194897b9c5b27e5d3c65.
